website/tests/acceptance/components/stepper/nav-test.js

```diff
@@ -8,17 +8,17 @@ import { visit, currentURL } from '@ember/test-helpers';
 import { setupApplicationTest } from 'website/tests/helpers';
 import { a11yAudit } from 'ember-a11y-testing/test-support';
 
-module('Acceptance | components/stepper', function (hooks) {
+module('Acceptance | components/stepper/nav', function (hooks) {
```

Comment:
Issue: This test should exist for Stepper Nav, stepper list, and stepper indicator.